patch/kernel/archive/rockchip64-6.6/dt/rk3566-orangepi-3b-sata.dts

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
```

Comment:
Do we really need new devicetree? I think enabling overlay for who needs sata should be enough.

Reply:
Oh didn't see the comment. It makes sense